build-tests/localization-plugin-test-01/.heft/webpack.json

```diff
@@ -0,0 +1,3 @@
+{
+  "webpackConfigFilePath": "./webpack.config.js"
```

Comment:
Is `localization-plugin-test-01` the test for the webpack plugin?  That is not very obvious.

It might be more intuitive to find the localization plugin test case inside `heft-webpack-test`.

Or perhaps it would be better to simply have separate test projects, so we can do complex localization scenarios in `localization-plugin-test`, and more basic webpack tests in `heft-webpack-test`.

Speaking of which, instead of `heft-test-01`, maybe the names should start more like this:
- `heft-web-test`
- `heft-node-test`

...and add `-01` later when we have more than one of a given scenario.